tests/test_sqlDatabaseDict.py

```diff
@@ -83,6 +85,24 @@ def testKeyNotInValue(self):
         d = self.registry.makeDatabaseDict(table="TestTable", key=self.key, types=self.types, value=value)
         self.checkDatabaseDict(d, data)
 
+    def testLengths(self):
+        """Test that when a length is specified that it is actually used."""
+        value = namedtuple("TestValue", ["y", "z"])
+        data = {
+            0: value(y="zero", z=0.0),
+            1: value(y="this is too long", z=0.1),
```

Comment:
My only comment is that it might be nice to name these keys "passing" and "failing" or something like that rather than generic 0 and 1 magic numbers, but its in test code, and I dont feel very strongly about it.